charts/newrelic-k8s-metrics-adapter/templates/cert-manager.yaml

```diff
@@ -0,0 +1,53 @@
+{{ if .Values.certManager.enabled }}
+---
+# Create a selfsigned Issuer, in order to create a root CA certificate for
+# signing webhook serving certificates
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: {{ include "newrelic-k8s-metrics-adapter.fullname.self-signed-issuer" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  selfSigned: {}
+---
+# Generate a CA Certificate used to sign certificates for the webhook
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ include "newrelic.common.naming.fullname" . }}-root-cert
```

Comment:
I see you declared `{{- define "newrelic-k8s-metrics-adapter.fullname.root-issuer" -}}` but you are not using it here so this hit the limit of chars of a DNS name the the function `newrelic.common.naming.truncateToDNSWithSuffix` solves.